addons/mail/models/ir_websocket.py

```diff
@@ -75,6 +79,25 @@ def _prepare_subscribe_data(self, channels, last):
         data["missed_presences"] = self.env["mail.presence"].sudo().search(presence_domain)
         return data
 
+    def _build_bus_channel_list(self, channels):
+        chatter_channel_ids = {}
+        regex = re.compile(r"thread-([-_\w\.]*)_(\d+)")
+        for channel in channels:
+            if isinstance(channel, str):
+                if match := re.match(regex, channel):
+                    model, id = match.groups()
+                    if model not in self.env:
+                        _logger.warning("Trying to build invalid model %s (%s) for bus channel list", model, channel)
+                        continue
+                    channels.remove(channel)
+                    chatter_channel_ids.setdefault(model, set()).add(int(id))
+        for model, ids in chatter_channel_ids.items():
+            threads = self.env[model].search([("id", "in", list(ids))])
```

Comment:
If we intend to allow portal users to benefit from live changes too, we'll also need to grant access with access token, and not with just search/ACL.